ee/hogai/chat_agent/query_executor/query_executor.py

```diff
@@ -117,7 +105,7 @@ def __init__(self, team: Team, utc_now_datetime: datetime):
 
     async def arun_and_format_query(
         self,
-        query: SupportedQueryTypes,
+        query: AnyPydanticModelQuery | AnyAssistantGeneratedQuery,
```

Comment:
same here, let's just substitute this union everywhere with a `AnyAgentSupportedQuery` or something